client/src/components/related/RelatedProducts.jsx

```diff
@@ -178,19 +85,19 @@ class RelatedProducts extends React.Component {
     const { selectedProduct, selectProduct, checkCache, updateCache } = this.props;
 
     return (
-      <div id='RelatedProdcuts'>
+      <div id='RelatedProducts'>
```

Comment:
for css-styling convention, should be `related-products`.

Reply:
For ids of React components, I use the same name, as in "App".